source/uwp/Renderer/lib/AdaptiveFact.h

```diff
@@ -36,6 +39,9 @@ AdaptiveNamespaceStart
 
     private:
         std::shared_ptr<AdaptiveSharedNamespace::Fact> m_sharedFact;
```

Comment:
>std::shared_ptr<AdaptiveSharedNamespace::Fact> m_sharedFact; [](start = 8, length = 60)

You don't need this to be a member variable anymore now that we're storing all the properties in their own members.